docs/pages.jl

```diff
@@ -45,8 +45,8 @@ pages = Any[
         "steady_state_functionality/dynamical_systems.md"
     ],
     "Inverse problems" => Any[
-        "inverse_problems/optimization_ode_param_fitting.md",
         "inverse_problems/petab_ode_param_fitting.md",
```

Comment:
This feels like the simpler to use and more reliable option, so figured it might make sense to put first. Then if someone cannot use it they can proceed to the next option.